ios/Classes/ScreenLockCheckPlugin.m

```diff
@@ -16,7 +16,9 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
    NSError *error;
    BOOL isLockScreenEnabled = [context canEvaluatePolicy:LAPolicyDeviceOwnerAuthentication error:&error];
    if (error != nil) {
-     result([FlutterError errorWithCode: @“UNAVAILABLE” message: @“Local authentication context is unavailable” details: nil]);
+       result([FlutterError errorWithCode:@"UNAVAILABLE"
```

Comment:
As you can notice these quotes were not the ones iOS liked.

Reply:
And this is what happens when you copy and paste from the internet...